pkg/services/rocketchat/rocketchat.go

```diff
@@ -30,18 +29,28 @@ func (service *Service) Initialize(configURL *url.URL, logger *log.Logger) error
 
 // Send a notification message to Rocket.chat
 func (service *Service) Send(message string, params *types.Params) error {
+	var res *http.Response
+	var err error
 	config := service.config
 	apiURL := buildURL(config)
 	json, _ := CreateJSONPayload(config, message, params)
-	res, err := http.Post(apiURL, "application/json", bytes.NewReader(json))
+	res, err = http.Post(apiURL, "application/json", bytes.NewReader(json))
+	if err != nil {
+		return fmt.Errorf("Error while posting to URL: %v\nHOST: %s\nPORT: %s\n", err, config.Host, config.Port)
 
+	}
 	if res.StatusCode != http.StatusOK {
 		return fmt.Errorf("failed to send notification to service, response status code %s", res.Status)
 	}
 	return err
 }
 
 func buildURL(config *Config) string {
-	return fmt.Sprintf("https://%s/hooks/%s/%s", config.Host, config.TokenA, config.TokenB)
+	if config.Port != "443" && config.Port != "" {
```

Comment:
```suggestion
	if config.Port != "" {
```

No need to do an explicit check for 443. It shouldn't matter if it gets appended or not given that `https` is 443 anyway.

Reply:
DONE.